indico/MaKaC/services/implementation/registration.py

```diff
@@ -77,11 +78,9 @@ def _handleGet(self):
 class ConferenceEticketQRCode(RegistrationModifBase):
 
     def _getAnswer(self):
-
-        consumers = dict((consumer.getName(), consumer) for consumer in ConsumerHolder().getList())
-
-        if "indico-checkin" not in consumers:
-            raise NoReportError(_("There is no indico-checkin consumer key for OAuth"))
+        oauth_checkin = OAuthApplication.find_first(name='Indico Checkin')
```

Comment:
Ugh... :cancer:

I'd rather add a config option where you specify the ID (or name, but I'd go for ID) of the OAuth app that's used for checkin.